docs/source/deploypeer/peerchecklist.md

```diff
@@ -0,0 +1,398 @@
+#  Checklist for a production peer
+
+As you prepare to build a production peer, you need to customize the configuration by editing the [core.yaml](https://github.com/hyperledger/fabric/blob/{BRANCH}/sampleconfig/core.yaml) file, which is copied into the `/config` directory when downloading the Fabric binaries, and available within the Fabric peer image at `/etc/hyperledger/fabric/core.yaml`.
+
+While in a production environment you could override the environment variables in the `core.yaml` file in your Docker container or your Kubernetes job, these instructions show how to edit `core.yaml` instead. It’s important to understand the parameters in the configuration file and their dependancies on other parameter settings in the file. Blindly overriding one setting using an environment variable could affect the functionality of another setting. Therefore, the recommendation is that before starting the peer, you make the modifications to the settings in the configuration file to become familiar with the available settings and how they work. Afterwards, you may choose to override these parameters using environment variables.
+
+This checklist covers key configuration parameters for setting up a production network. Of course, you can always refer to the core.yaml file for additional parameters or more information. It also provides guidance on which parameters should be overridden. The list of parameters that you need to understand and that are described in this topic include:
+
+- [peer.id](#peerid)
+- [peer.networkId](#peernetworkid)
+- [peer.listenAddress](#peerlistenaddress)
+- [peer.chaincodeListenAddress](#peerchaincodelistenaddress)
+- [peer.chaincodeAddress](#peerchaincodeaddress)
+- [peer.address](#peeraddress)
+- [peer.mspConfigPath](#peermspconfigpath)
+- [peer.localMspId](#peerlocalmspid)
+- [peer.fileSystemPath](#peerfilesystempath)
+- [peer.gossip.x](#peergossipx)
```

Comment:
Not sure if the 'x' was meant to be a wildcard * ?  If you want to indicate categories of properties versus individual properties, I would use `.*`

```suggestion
- [peer.gossip.*](#peergossipx)
```